Dockerfile

```
@@ -1,36 +1,42 @@
-FROM ghcr.io/astral-sh/uv:python3.13-bookworm
+# Lightweight multi-stage build for container optimization
+# Stage 1: Build dependencies using uv
+FROM ghcr.io/astral-sh/uv:python3.13-bookworm AS builder
```

```suggestion
FROM ghcr.io/astral-sh/uv:python3.13-alpine AS builder
```

Pythonのバイトコードをコンパイルするのでビルダーも最終イメージのOSと合わせてください